src/components/FormResponse.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const FormResponse = ({ msgs, showIfEmpty = false, isError }) => {
```

Comment:
I thought i saw on the front-end one time some html that seemed to do what this component does, but couldn't find anything in components folder. Do you know if anything? Just prints messages in an outlined box basically.

Reply:
Generally for this I've used https://github.com/elmadev/elmaonline-web/blob/dev/src/components/Feedback.js

Reply:
Oh ok. Could always re-factor to that in the future if it seems worthwhile. Would be more consistent, but also the new feature is a tiny one that not many people will even be likely to use IMO.